### DISCUSSION TITLE
Welcome to Our-E-School Discussions!

### DISCUSSION THREAD

**Discussion by ketanchoyal** [ketanchoyal/Our-E-School#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bustiloo** [comment]
Flutter beginner here.
Thanks for your effort.. well done!

**Reply by ketanchoyal** [reply]
Your Welcome, Glad my project helped

**Comment by TurboTronix** [comment]
Hi there, 

This looks promising, I am currently looking for a starting point in order to create a daycare system. Was wondering if you'll be adding the sql integration. I wanted to start playing with this but without the sql part it's a PITA...I want to add RFID integration, a key part of what I'm looking for. I got the hardware working just need to connect it to a hosted db.

**Reply by ketanchoyal** [reply]
Sorry @TurboTronix, integration of SQL will take a lot of time which I currently lack. Maybe in future

**Reply by TurboTronix** [reply]
Yeah I've looked into this and both aren't the same, one is relational the other is not.

I'm looking into firebase, seems like it's enough for my needs.

**Comment by Sormox** [comment]
I'm Beginner to android studio and Githup . How I can import the project to my android studio without errors ?
